packages/account/src/Sections/Profile/PhoneVerification/phone-number-verified-modal.tsx

```diff
@@ -61,7 +59,8 @@ const PhoneNumberVerifiedModal = observer(({ should_show_phone_number_verified_m
                 <div className='phone-verification__verified-modal--contents'>
                     <div className='phone-verification__verified-modal--contents__phone-number-container'>
                         <div className='phone-verification__verified-modal--contents__phone-number-container__phone-number'>
-                            {phone}
+                            {/* @ts-expect-error will remove once solved */}
+                            {`${account_settings?.calling_country_code}${account_settings?.phone}`}
```

Comment:
<Localize
                                            i18n_default_text='{{phone}} is verified as your phone number.'
                                            values={{
                                                phone: `${account_settings?.calling_country_code}${account_settings?.phone}`,
                                            }}
                                        />

                                        cant we use Localize like this?

Reply:
@amina-deriv i did it this way because in RTL arabic translation, the + sign alignment will not be correct.